content/sensu-go/5.15/guides/use-federation.md

```diff
@@ -0,0 +1,344 @@
+
+## Configure federation
+
+Complete federation of multiple Sensu instances relies on a combination of features:
+
+| Feature                             | Purpose in federation |
+|-------------------------------------|--------------------------------------------------------------------|
+| JSON Web Token (JWT) authentication | Cross-cluster token authentication using asymmetric key encryption |
+| etcd replicators                    | Replicate RBAC policy across clusters and namespaces |
+| Federation Gateway and APIs         | Configure federation access for cross-cluster visibility in web UI |
+
+The following steps are required to configure these features for unified visibility in a single web UI. Our examples assume that we wish to federate three named Sensu clusters:
+
+| Cluster name | Hostname |
+|--------------|-----------------------|
+| `gateway` | sensu.gateway.example.com |
+| `alpha` | sensu.alpha.example.com |
+| `beta` | sensu.beta.example.com |
+
+In this example, the `gateway` cluster will be the entry point for operators to manage Sensu resources in the `alpha` and `beta` clusters.
+
+This diagram depicts the federation relationship documented in this guide:
+
+<!-- Federation diagram source: https://www.lucidchart.com/documents/edit/1b676df9-534e-40e4-9881-6313013ecd28/n~8S.VTyl5JQ -->
+<a href="/images/federation-guide-diagram.png"><img alt="Diagram depicting this guide's example federation architecture" src="/images/federation-guide-diagram.png" width="800 px"></a>
+
+### Step 1 Configure backends for TLS
+
+Because federation depends on communication with multiple disparate clusters, working TLS is required for successful configuration. This guide assumes that you have provided each backend member with TLS credentials (key and certificate), as well as the CA certificate chain required for one Sensu backend to validate the certificates presented by other backends. If you don't have existing infrastructure for issuing certificates, see [Securing Sensu][13] for our recommended self-signed certificate issuance process.
+
+This prerequisite extends to configuring the following Sensu backend etcd parameters:
+
+| Backend property             | Note |
+|------------------------------|------|
+| `etcd-cert-file`             | Path to certificate used for TLS on etcd client/peer communications.  |
+| `etcd-key-file`              | Path to key corresponding with `etcd-cert-file` certificate. |
+| `etcd-trusted-ca-file`       | Path to CA certificate chain file. This CA certificate chain must be usable to validate certificates for all backends in the federation. |
+| `etcd-client-cert-auth`      | Enforces certificate validation to authenticate etcd replicator connections. We recommend setting to `true` |
+| `etcd-advertise-client-urls` | List of https URLs to advertise for etcd replicators, accessible by other backends in the federation (e.g. `https://sensu.beta.example.com:2378`) |
```

Comment:
Should the port here be `2379`? Using `2378` didn't work as that is not a port used by Sensu Go.

It does work with `2379`, but I wasn't sure if using the host and all interfaces for both `etcd-advertise-client-urls` and `etcd-listen-client-urls` would cause issues. Or if them being websockets(?) it is ok.